services/minespace-web/src/components/Forms/projects/informationRequirementsTable/ReviewSubmitInformationRequirementsTable.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Checkbox, Col, Collapse, Input, Typography } from "antd";
+import customPropTypes from "@/customPropTypes";
+
+const { Panel } = Collapse;
+const { Title } = Typography;
+
+const propTypes = {
+  requirements: customPropTypes.requirements.isRequired,
+  sub_requirement: customPropTypes.subRequirements.isRequired,
+};
+
+const renderSubrequirement = (item) => item.map((it) => <Subrequirement sub_requirement={it} />);
+
+const ReviewSubmitInformationRequirementsTable = ({ requirements }) => {
+  return (
+    <Collapse defaultActiveKey={["1"]}>
+      <br />
+      <Panel header={<Title level={5}>{`${requirements?.description}`}</Title>} key="1">
+        {requirements !== undefined ? renderSubrequirement(requirements?.sub_requirements) : null}
+      </Panel>
+    </Collapse>
+  );
+};
+
+// Apply font size depending the level of requirement description:
+// 1.1 => 16px bold. , 1.2.1 => 14px bold, 1.2.1.1 => 14 px
+const DescriptionTextSytle = (step) => {
+  if (step.match(/\./g).length === 2) {
+    return { fontWeight: "bold", fontSize: "16px" };
+  } if (step.match(/\./g).length === 3) {
+    return { fontWeight: "bold", fontSize: "14px" };
+  } 
+    return { fontSize: "14px" };
+  
+};
+
+const Subrequirement = ({ sub_requirement }) => (
+  <>
+    <Col span={24}>
+      <span id="description" style={DescriptionTextSytle(sub_requirement.step)}>
+        {`${sub_requirement.step} ${sub_requirement.description}`}
+      </span>
+
+      {sub_requirement.required === undefined && sub_requirement.methods === undefined ? null : (
+        <>
+          <p>
+            <Checkbox checked={sub_requirement.required.toLowerCase() === "true"}>
```

Comment:
Is there a reason the boolean is a string here?

Reply:
Fixing response model in backend to return Boolean.